Tools/autotest/arducopter.py

```diff
@@ -1244,6 +1244,57 @@ def fly_vision_position(self):
         if ex is not None:
             raise ex
 
+    def fly_nav_delay(self):
+        '''fly a simple mission that has a delay in it'''
```

Comment:
"""

Reply:
Why does it matter?

Reply:
For autodoc... that we don't support yet. And to be compliant with pep8 style